lib/Cake/Test/Case/View/Helper/PaginatorHelperTest.php

```diff
@@ -2829,6 +2928,12 @@ public function testCurrent() {
 	public function testNoDefaultModel() {
 		$this->Paginator->request = new CakeRequest(null, false);
 		$this->assertNull($this->Paginator->defaultModel());
+
+		$this->Paginator->defaultModel('Article');
```

Comment:
I would move this to a separate test function, cause now `testNoDefaultModel()` name doesn't match what's being tested.